src/components/menu/Menu.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { Pressable, StyleSheet, View, ViewStyle } from 'react-native';
+import { Menu as BaseMenu } from 'react-native-paper';
+import { useTheme } from '../../styles/themes';
+import { Icon } from '../icons/Icon';
+import type { IconName } from '../icons/IconProps';
+
+type MenuOption = {
+    title: string;
+    onPress: () => void;
+};
+
+interface Props {
+    visible: boolean;
+    openMenu: () => void;
+    closeMenu: () => void;
+    options: MenuOption[];
+    menuIcon: IconName;
```

Comment:
```suggestion
    icon: IconName;
```
In a menu component im not sure we need the 'menu' prefix

Reply:
fixed!